front/pages/w/[wId]/subscription/index.tsx

```diff
@@ -207,6 +280,39 @@ export default function Subscription({
       topNavigationCurrent="admin"
       subNavigation={subNavigationAdmin({ owner, current: "subscription" })}
     >
+      <CancelFreeTrialDialog
+        show={showCancelFreeTrialDialog}
+        onClose={() => setShowCancelFreeTrialDialog(false)}
+        onValidate={async () => {
+          cancelFreeTrial()
```

Comment:
Why not having `onValidate=cancelFreeTrial' and have it manage the state for setShowCancelFreeTrialDialog. I'm not sure also why we console.error here when we manage the error directly in cancelFreeTrial? 🙏🏻